app/Resources/assets/js/lsdoc/index.js

```diff
@@ -631,6 +631,8 @@ var Util = (function () {
     };
 })();
 
+apx.Util = Util;
```

Comment:
Setting `apx.Util` fails on pages that do not have `apx` defined (such as the front page).